spring-integration-core/src/test/java/org/springframework/integration/config/AggregatorParserTests-context.xml

```diff
@@ -44,7 +47,7 @@
 		scheduler="scheduler"
 		message-store="store"
 		order="5">
-			<expire-advice-chain/>
+			<expire-transactional/>
```

Comment:
??? Not sure in your question.
This functionality hasn't been changed. 
The `<expire-advice-chain/>` still exists. I just didn't want to write some class for mock `Advice` and just used `transactionManager` on the matter. 
The issue with test that it checks `adviceChain` presence, but now we don't populate empty `adviceChain`.

Am I missing anything else ?